recipe/meta.yaml

```diff
@@ -66,12 +66,13 @@ requirements:
     - subprocess32  # [py2k and unix]
     - icu 58.*  # [linux]
     - tornado
+    - kiwisolver
 
 test:
   imports:
     - matplotlib
     - matplotlib.pyplot
-    - matplotlib._cntr
+    # - matplotlib._cntr
```

Comment:
@tacaswell this test failed. Was `matplotlib._cntr` it removed or am I missing something?

Reply:
It was removed, also had to remove it from the rc branch.

Reply:
Thanks. I'll remove the commented out test in another PR.